apps/gyrokinetic.c

```diff
@@ -136,12 +136,13 @@ gkyl_gyrokinetic_app_new(struct gkyl_gk *gk)
   struct gkyl_rect_grid geo_grid;
   struct gkyl_range geo_local;
   struct gkyl_range geo_local_ext;
+  struct gkyl_range geo_global;
+  struct gkyl_range geo_global_ext;
   struct gkyl_basis geo_basis;
-  bool geo_3d_use_gpu = app->use_gpu;
+  bool geo_3d_use_gpu = false;
 
   if(app->cdim < 3){
-    geo_grid = agument_grid(app->grid, gk->geometry);
-    gkyl_create_grid_ranges(&geo_grid, ghost, &geo_local_ext, &geo_local);
+    geo_grid = augment_grid(app->grid, gk->geometry);
     geo_3d_use_gpu = false;
```

Comment:
Let's delete this line now that geo_3d_use_gpu is now always false and let's add a comment that we always initialize geometry on host.